iarc_forebrain/scripts/Drone.py

```diff
@@ -0,0 +1,518 @@
+import math
+
+import rospy
+import tf
+import tf.transformations
+from ardrone_autonomy.msg import Navdata
+from geometry_msgs.msg import Twist, PoseStamped, Pose, Point, Quaternion
+from iarc_arbiter.msg import RegisterBehavior, VelAlt
+from iarc_main.msg import Roomba
+from numpy import pi
+from std_msgs.msg import String, Empty, Header
+
+
+class Drone:
+    ROOMBA_HEIGHT = 0.1  # Height of a Roomba top in meters
+    MAX_VERTICAL_VEL = 2.0
```

Comment:
These parameters are currently not doing much, and is possibly shadowing a configurable max_vertical_vel parameter under ddyn_rec